pkgs/development/python-modules/mnist/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage, fetchPypi, isPy27, lib, mock, numpy }:
+
+buildPythonPackage rec {
+  pname = "mnist";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m3vz9z7gay8nndid2784j95fzif502shfvik3f8mggvx6nlz4r4";
+  };
+
+  propagatedBuildInputs = [
+  numpy
+  ] ++ lib.optionals (isPy27) [ mock ];
+
+  # There exist no tests
```

Comment:
There are tests: https://github.com/datapythonista/mnist/tree/mnist-0.2.2/tests. However, they're probably not distributed on PyPI.